### ISSUE TITLE
Height and width inversion

### ISSUE THREAD

**Issue by zone47** [tategallery/collection#35]
Hello,
Thanks for the dataset. After looking in some records, it seems that there is an inversion with width and height in JSON collection records.
Best regards

**Comment by sebastfr** [comment]
@zone47 thank you for your feedback - do you have any examples please?

**Comment by zone47** [comment]
here : https://github.com/tategallery/collection/blob/master/artworks/p/015/p01505-11583.json
or here : https://github.com/tategallery/collection/blob/master/artworks/t/005/t00505-13260.json
All records with height and width, actually.
For example :
http://www.tate.org.uk/art/artworks/turner-a-standing-male-nude-with-right-arm-raised-seen-from-behind-d00202
https://github.com/tategallery/collection/blob/master/artworks/d/002/d00202-27618.json
"dimensions": "support: 514 x 349 mm", 
"height": "349"
"width": "514"
It's the opposite : height 514mm and width 349mm
For " 514 x 349 mm", in art history it is usual to set height before width.

**Comment by sebastfr** [comment]
@zone47 you are correct, i suspect our script generating the data got it wrong - i will try to schedule to have the data updated. Again, thank you for getting in touch, much appreciated!

**Comment by zone47** [comment]
Thank you for your reactivity! It's a great dataset (perimeter of the collection and quality). There is always some things to improve or correct but it is already very useful and I'm very happy that it is reused ( https://www.wikidata.org/wiki/Wikidata:WikiProject_sum_of_all_paintings/Collection/Tate ).

**Comment by sebastfr** [comment]
@zone47 and thanks for sharing that link, that's really cool! - we do use wikidata's data to hook up some of "Tate artists" to other datasets :-) this is _just_ awesome.

**Comment by multichill** [comment]
Tate artists, right, we still have to match a couple of them, see https://www.wikidata.org/wiki/Wikidata:Database_reports/Complex_constraint_violations/P2741

BTW, the program I used to add the paintings is at https://github.com/multichill/toollabs/blob/master/bot/wikidata/tate_import.py

**Comment by sebastfr** [comment]
@multichill I'm happy to help if you think i can contribute (also we could work on a fresher batch of data). let me know?

**Comment by multichill** [comment]
Update of the data would be really nice! That would enable me to fix the swapped width/height on our side and maybe add some new works.
